src/app/home/avatar-generator/selector/atribute-selector.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit, Input, Inject } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import {AvatarDecoratorItem, AvatarOptions} from '../Avatartypes';
+import {buildAvatar} from '../AvatarUtils';
+import {getPreview} from '../AvatarSelectorPath';
+
+@Component({
+  // tslint:disable-next-line: component-selector
+  selector: 'atribute-selector',
+  templateUrl: './atribute-selector.html',
+  styleUrls: ['./atribute-selector.scss'],
+})
+export class AtributeSelector implements OnInit {
+  // @Output() newValue = new EventEmitter<boolean>();
```

Comment:
I think we can safely remove this commented line if it's not going to be used

Reply:
fixed